### PR TITLE
Refactored the code related to the movie objects

### PR DESCRIPTION
**Pull request by PurtucRobert**
- Refactored the code related to the movie objects
- Fixed the time format

### CODE REVIEW

File: cinemy/cinemy/settings.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "core",
     "contact",
     "login",
+    "core.templatetags.filter",
```
Comment (ghost) [inline cinemy/cinemy/settings.py]: why did you add this here?

**Review by drejleeo — COMMENTED** [review]
.

File: cinemy/cinemy/settings.py (hunk 1 above)
Comment (PurtucRobert) [reply cinemy/cinemy/settings.py:45]: I have created a custom filter which is transforming the time from integer value in minutes to H:m format.

Comment (PurtucRobert) [reply cinemy/cinemy/settings.py]: Removed it as it's not needed.

**Review by drejleeo — APPROVED** [review]
